src/gui/gui_topbar.hpp

```diff
@@ -408,6 +519,21 @@ class topbar_rebels_icon : public standard_nation_button {
 			text::localised_format_box(state, contents, box, std::string_view("countryalert_no_haverebels"), text::substitution_map{});
 		} else if(rebellions_iter.begin() != rebellions_iter.end()) {
 			text::localised_format_box(state, contents, box, std::string_view("countryalert_haverebels"), text::substitution_map{});
+			auto nation_fat_id = dcon::fatten(state.world, nation_id);
+			nation_fat_id.for_each_rebellion_within([&](dcon::rebellion_within_id rbl) {
+				auto fat_id = dcon::fatten(state.world, rbl);
+				auto rbl_fact_fat_id = fat_id.get_rebels();
+				auto rbl_type_fat_id = rbl_fact_fat_id.get_type();
+				auto rbl_fact_slim_id = rbl_fact_fat_id.id;
+				text::add_line_break_to_layout_box(contents, state, box);
+				text::substitution_map sub;
+				auto rebelname = text::produce_simple_string(state, rbl_type_fat_id.get_name());
+				auto rebelsize = text::prettify(rebel::get_faction_brigades_active(state, rbl_fact_slim_id));
+				text::add_to_substitution_map(sub, text::variable_type::name, rebelname);
+				text::add_to_substitution_map(sub, text::variable_type::strength, rebelsize);
+				text::add_to_substitution_map(sub, text::variable_type::org, text::format_percentage(rebel::get_faction_organization(state, rbl_fact_slim_id)));
```

Comment:
This is one of those problematic places where a temporary string is created and then destroyed while a pointer to it in the substitution map still exists